source/stylesheets/navigation.sass

```diff
@@ -217,20 +219,28 @@ header
   .header-page-link
     a, .category-nav-heading
       display: flex
+      color: var(--header-text-color)
       padding: 4px 0
       position: relative
       outline-offset: 4px
       border-radius: 0
 
-      &:hover
+      &:hover, &:focus
         color: var(--link-hover-color)
 
     a:hover
       @media (hover: hover)
         text-decoration: underline
         text-decoration-thickness: 2px
+    
+    svg
+      fill: var(--header-text-color)
```

Comment:
as long as a parent element has a color defined (and a hover color defined) then svgs can get `fill: currentColor` either in the CSS or as an attribute on the svg element itself